From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Refrag <R3FR4GYT@gmail.com>
Date: Sun, 4 Feb 2024 17:45:15 +0100
Subject: [PATCH] Fix libgme include

One of the build targets is looking for the gme.h header.
It seems like it cannot automatically find it so we help it a little.
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 80a3bdcd6..a01d32e1f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -168,6 +168,7 @@ set(SRB2_SDL2_EXE_NAME "" CACHE STRING "Override executable binary output name")
 set(SRB2_SDL2_EXE_SUFFIX "" CACHE STRING "Optional executable suffix, separated by an underscore")
 
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/src)
+include_directories(${CMAKE_BINARY_DIR}/_deps/libgme-src)
 
 add_subdirectory(src)
 add_subdirectory(assets)
